conf/application.conf

```diff
@@ -162,6 +162,10 @@ mongo-async-driver {
 features {
   check-cesa-relationships = false
   copy-cesa-relationships = false
+
+  inactive-relationships {
```

Comment:
`features` container is for on/off switches only

Reply:
👍